### PR TITLE
ERR: better error message with invalid argument in sort_values

### PR DESCRIPTION
**Pull request by ghost**
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] Ensure all linting tests pass, see [here](https://pandas.pydata.org/pandas-docs/dev/development/contributing.html#code-standards) for how to run them
- [ ] whatsnew entry
added a if branch to check the ascending type, as the existing one accepts value with string as well instead of boolean.

k=df.sort_values(by=['col1'],ascending="False") does not give exception with str.
added piece of code to raise exception if the type is not boolean

### CODE REVIEW

File: pandas/core/series.py [hunk 1]
```diff
@@ -3441,8 +3441,10 @@ def sort_values(
                 )
             ascending = ascending[0]
 
-        if not is_bool(ascending):
-            raise ValueError("ascending must be boolean")
+        #if not is_bool(ascending):
```
Comment (jbrockmendel) [inline pandas/core/series.py:3444]: is_bool(ascending) is equivalent to `isinstance(ascending, (bool, np.bool_)`, so this shouldn't make a difference.  if im wrong and this does make a difference, this needs a test

File: pandas/core/series.py [hunk 2]
```diff
@@ -3441,8 +3441,10 @@ def sort_values(
...
+        #    raise ValueError("ascending must be boolean")
```
Comment (jbrockmendel) [inline pandas/core/series.py:3445]: pls dont leave commented-out code, just delete it

File: pandas/core/series.py [hunk 3]
```diff
@@ -3441,8 +3441,10 @@ def sort_values(
...
+        if type(ascending)!=bool:
+            raise ValueError("ascending should of type bool")
```
Comment (jbrockmendel) [inline pandas/core/series.py:3447]: are there some words left out of this message?  what's wrong with the existing one?

File: pandas/core/series.py (hunk 1 above)
Comment (deej197) [reply pandas/core/series.py:3444]: i see this issue open #41634. if this is not resolved, then is_bool function is not working as expected.

regarding the comment, wasnt aware of the same as this is my first contribution. will note this for future.

Comment (lithomas1) [reply pandas/core/series.py]: @deej197 Isn't this issue for DataFrame not series? 
https://github.com/pandas-dev/pandas/blob/dad3e7fc3a2a75ba5f330899be0639cff0f73f6c/pandas/core/frame.py#L6208

You might want to look into using ``validate_ascending``
https://github.com/pandas-dev/pandas/blob/dad3e7fc3a2a75ba5f330899be0639cff0f73f6c/pandas/util/_validators.py#L421

Comment (deej197) [reply pandas/core/series.py]: @lithomas1 I can make the changes in this validate_ascending function , but i am stuck in running tests locally. I tried replacing the pandas package in C:\Users\usename\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.7_qbz5n2kfra8p0\LocalCache\local-packages\Python37\site-packages with the modified package, but i dont see the changes being applied.

appreciate any suggestions on executing tests internally.

**Comment by datapythonista** [comment]
Thanks for the contribution @deej197, but unfortunately you won't be able to contribute with a proper development environment. If you could run this test locally you could see that:
- It's adding new bugs, and doesn't affect the current one
- The problem is not the one you mention in the title, the error message is fine, the problem is that it only raises on `Series`, and not in `DataFrame`.

And it's also better to start by opening an issue, finding a reproducible example, and when there is agreement with the solution, then open the PR (which would require tests and a release note, besides the code changes). I opened an issue myself, so you can see how would be better to start: #42434. Feel free to work on the issue, but as I said, you need a working development environment. Otherwise you won't be able to test what you're doing, and you'll be submitting obviously wrong PRs like this, that don't add value, and waste reviewers time.

If you follow the contribution guide, you shouldn't have problems setting up the develoment environment, if something is not clear enough, feel free to open an issue to ask for clarification in the docs.

**Pull request closed without merging**